### PR TITLE
Fix broken integration tests

### PR DESCRIPTION
**Pull request by florianGla**
The integration tests now work on windows.

### CODE REVIEW

File: pkg/runfiles/finder_test.go [hunk 1]
```diff
@@ -24,38 +26,49 @@ func TestMain(m *testing.M) {
 	m.Run()
 }
 
-func TestLlvmToolPath(t *testing.T) {
+func TestIntegration_LlvmToolPath(t *testing.T) {
```
Comment (MarkusZoppelt) [inline pkg/runfiles/finder_test.go:29]:
```suggestion
func TestIntegration_LLVMToolPath(t *testing.T) {
```

File: internal/cmd/coverage/llvm/llvm_test.go [hunk 2]
```diff
@@ -92,8 +101,6 @@ func TestLLVM(t *testing.T) {
 				assert.DirExists(t, reportPath)
 				assert.FileExists(t, filepath.Join(reportPath, "index.html"))
 			}
-			assert.Contains(t, bOut.String(), "src/explore_me.cpp")
-			assert.Contains(t, bOut.String(), "my_fuzz_test.cpp")
```
Comment (MarkusZoppelt) [inline internal/cmd/coverage/llvm/llvm_test.go:96]: why were these removed?

**Review by MarkusZoppelt — APPROVED** [review]

File: internal/cmd/coverage/llvm/llvm_test.go (hunk 2 above)
Comment (florianGla) [reply internal/cmd/coverage/llvm/llvm_test.go:96]: They check if those lines appear in the build output of the makefile. I changed the build system to cmake, in order to be able to test it on Windows. Those checks cannot be done for cmake projects
